translations/wesnoth-A_New_Order/cs.po

```diff
@@ -24180,7 +24090,7 @@ msgid ""
 "What part of the phrase 'do not kill their leaders, leave them for me' was "
 "so difficult to understand?"
 msgstr ""
-"Čemu z věty 'nezabíjejte jejich velitele, nechte je mně' jsi nerozuměl?"
+"Čemu z věty: „Nezabíjejte jejich velitele, nechte je mně,“ jsi nerozuměl?"
```

Comment:
```suggestion
"Čemu z věty „Nezabíjejte jejich velitele, nechte je mně.“ jsi nerozuměl?"
```

Reply:
Tak toto bych potřeboval vysvětlit. Nepochybuji o tom, že to máš správně, ale potřebuji tomu porozumět. Ta dvojtečka a čárka není správně obecně či proto, že tvar originálu nemá tu větu „vloženou“?

Reply:
Já tam nevidím [uvozovací větu](https://www.google.com/search?q=%22v%C4%9Bta+uvozovac%C3%AD%22) ani přímou řeč, takže ta dvojtečka před prvními uvozovkami a čárka před druhými mi přijdou navíc. Věta bez citace dává smysl sama o sobě:

Čemu z (té) věty jsi nerozuměl?

Řekl bych, že se zde jedná o případ z šestého odstavce (Jestliže se do výpovědi začleňuje část jiného textu...) [článku o uvozovkách](https://prirucka.ujc.cas.cz/?id=162), jen tady u nás to není na konci ani na začátku, ale uprostřed.

Chtěl jsem respektovat tvé velké písmeno **N** a původní překlad phrase -> věta, a tak doplnil na konec tečku, aby byla citována celá věta. To se ve výše zmíněném článku o uvozovkách píše ve čtvrtém odstavci.

Sám bych ale zachoval formu originálu i původního překladu, tedy malé **n** a bez interpunkčního znaménka před druhou uvozovkou. I proto, že v předchozím textu ta věta vypadá jinak, takže tu není citována přesně, aby bylo třeba zachovat její celistvost (s velkým písmenem na začátku). Vnímám, že je zde citována spíše myšlenka a na tu mi malé písmeno stačí.

Reply:
The English original here probably needs copyediting itself, so I wouldn't worry about matching it too closely... let me add a translation note, at least...

Reply:
No stále se učím. Je pravda, že věta dává smysl i bez uvozovek, strkat tam přímou řeč bylo z mé strany trochu násilné. Není to vskutku citováno přesně, předchozí výzvy jsou v trochu jiném tvaru.
Teda nejvíc mě překvapila ta tečka uprostřed věty, nikdy bych ji tam „opticky“ nedal. Ale koukám, že existují ještě větší „zhůvěřilosti“.
Přikláním se tedy k tvaru s malým písmenem a bez druhé interpunkce. Padne mi to nejvíc do oka. Jdu to opravit.